__tests__/app.test.js

```diff
@@ -142,9 +142,11 @@ describe("GET /api/articles/:article_id", () => {
           votes: 100,
           comment_count: 11,
```

Comment:
comment_count shouldn't be in the original test. You should be creating a new test. If you find this one is failing, it's to do with the toEqual matcher which looks to match the object EXACTLY. Are there any other matchers we could use here?